runtime/commonTest/src/kotlinx/serialization/features/PolymorphismTest.kt

```diff
@@ -59,17 +59,9 @@ class PolymorphismTest {
     }
 
     @Test
-    fun testExplicit() {
+    fun testExplicit() { // TODO explicit what?
```

Comment:
Explicit instantiation like `PolymorphicSerializer(PolyDerived::class)` in arguments of `json.stringify`

Reply:
Thanks, renamed the test